tests/benchmarks/test_work_stealing.py

```diff
@@ -0,0 +1,108 @@
+import time
+
+import dask.array as da
+import distributed
+import numpy as np
+import pytest
+from coiled.v2 import Cluster
+from dask import delayed, utils
+from distributed import Client, wait
+from tornado.ioloop import PeriodicCallback
+
+
+@pytest.mark.xfail(reason="https://github.com/dask/distributed/issues/6573")
```

Comment:
We should also include this test in `distributed` once we have a fix for dask/distributed#6573

Reply:
```suggestion
```

Please remove the xfail. This test now succeeds although it's slow.

This is "fixed by https://github.com/dask/distributed/pull/7022 already and it includes this as a test case (or a version o fit)